include/document_graph/content.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <variant>
+#include <eosio/name.hpp>
+#include <eosio/asset.hpp>
+#include <eosio/time.hpp>
+#include <eosio/crypto.hpp>
+
+namespace hypha
+{
+    struct Content
+    {
+        typedef std::variant<eosio::name, std::string, eosio::asset, eosio::time_point,
+                             std::int64_t, eosio::checksum256>
+            FlexValue;
+
+    public:
+        Content();
+        Content(std::string label, FlexValue value);
+        ~Content();
+
+        const FlexValue getValue();
+        void setValue(FlexValue value);
+
+        const std::string getLabel();
+        void setLabel(std::string label);
```

Comment:
We can get rid of them

Reply:
I removed these from ```Content```. I revisited member visibility in ```Document``` and ```Edge``` classes.  Ideally, the persistence details would not need to be seen from outside the class, but it is useful from ```DocumentGraph``` because there are functions where I iterate based on certain indexes. 

But for ```Document```, I changed the members to be private and added getters.